### ISSUE TITLE
Is this required in version of dokku?

### ISSUE THREAD

**Issue by ankitrg** [bigboxsoftware/dokku-sidekiq#14]
Do I need to install this? or I can just specify the  worker in Procfile and everything will work?

**Comment by andyklimczak** [comment]
I don't think so, you should be able to specify a `worker` for sidekiq in your Procfile
